### PR TITLE
Updates the README with build instructions.

### PR DESCRIPTION
**Pull request by camilasan**
- Adds a shorter version of how to build the client but the extended version will be kept on the Wiki.
- Makes the instructions available if you clone the repo and work offline 

Thanks @tobiasKaminsky for pointing this out :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,8 +7,38 @@ with your computer.
 
 ### :hammer_and_wrench: How to compile the desktop client
 
-- Instructions for compiling the client on Linux, Windows and mac OS can be found at:
-[Wiki/How to compile the desktop client](https://github.com/nextcloud/desktop/wiki/How-to-compile-the-desktop-client).
+#### :memo: Step by step instructions
+
```
Comment (juliushaertl) [inline README.md]: Maybe we should add a link to the dependencies required to build here as well: 
https://github.com/nextcloud/desktop/wiki/System-requirements-for-compiling-the-desktop-client

File: README.md [hunk 2]
```diff
@@ -3,12 +3,48 @@
...
+    <img src="https://nextcloud.com/wp-content/themes/next/assets/img/clients/desktop/macsettings.png?x16328" alt="Desktop Client on Mac OS]">
+</p>
+
 ## :blue_heart: :tada: Contributing
 
 ### :hammer_and_wrench: How to compile the desktop client
 
-- Instructions for compiling the client on Linux, Windows and mac OS can be found at:
-[Wiki/How to compile the desktop client](https://github.com/nextcloud/desktop/wiki/How-to-compile-the-desktop-client).
+:building_construction: [System requirements](https://github.com/nextcloud/desktop/wiki/System-requirements-for-compiling-the-desktop-client) includes OpenSSL 1.1.x, QtKeychain, Qt 5.x.x and zlib.
```
Comment (camilasan) [inline README.md]: @juliushaertl  I added it here.

File: README.md [hunk 3]
```diff
@@ -3,12 +3,48 @@
 The :computer: Nextcloud Desktop Client is a tool to synchronize files from Nextcloud Server
 with your computer.
 
+<p align="center">
```
Comment (camilasan) [inline README.md]: I forgot to mention this on the commit message. @jancborchardt suggested to add this image.

Comment (jancborchardt) [reply README.md]: :)

**Pull request merged**